### ISSUE TITLE
New habitat information

### ISSUE THREAD

**Issue by peterdesmet** [trias-project/unified-checklist#61]
Examine what habitat information is requested by GRIIS.

**Comment by peterdesmet** [comment]
These are the taxa for which the habitat is updated: [updated_habitat.txt](https://github.com/trias-project/unified-checklist/files/4730971/updated_habitat.txt)

**Comment by peterdesmet** [comment]
As remarked by @timadriaens, Platyhelminthes and Hemiptera are all treated as host by GRIIS, but this is not correct for many species, e.g.:

> [Bipalium kewense](https://en.wikipedia.org/wiki/Bipalium_kewense) Moseley, 1878: terrestrial worm, not a parasite
> Caenoplana bicolor (Graff, 1899): terrestrial worm, not a parasite
> [Nysius huttoni ](https://en.wikipedia.org/wiki/Nysius_huttoni)White & F.B., 1878: terrestrial hemiptera, not a parasite

**Comment by timadriaens** [comment]
indeed, some strange habitat notions in there, Nysius hottoni is a terrestrial hemipteran, Coenoplana bicolor and Bipalium kewense are not intestinal parasites but are terrestrial flatworm (free living). Dendrocoelum romanodanubiale is also a free living freshwater species. It looks like they've assigned the "host" habitat quick and dirty to entire higher taxa but clearly there are mistakes in there.

Also, we do not consider the habitat "host" but rather for the indicators focus on the environment. So clearly GRIIS and our unified have a rather different interpretation of "habitat".
